taxi/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from taxi.views import index
+
+urlpatterns = [path("", index, name="index")]
```

Comment:
It's better to have urlpatterns open like this:
`
urlpatterns = [
    ...
]
`
then you can add elements to it more quickly and it becomes more readable.